documentation/docs/mcp/pscale-mcp.md

```diff
@@ -0,0 +1,66 @@
+---
+title: PlanetScale Extension
+description: Add PScale Extension to goose
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import YouTubeShortEmbed from '@site/src/components/YouTubeShortEmbed';
+import GooseDesktopInstaller from '@site/src/components/GooseDesktopInstaller';
+import CLIExtensionInstructions from '@site/src/components/CLIExtensionInstructions';
+
+
+This tutorial covers how to add the [PScale MCP Server](https://github.com/planetscale/cli?tab=readme-ov-file#mcp-server-integration) as a goose extension to enable you to inspect your databses and ingest query insights.
+
+## Configuration
+
+:::info
+Note that you'll need [PScale](https://github.com/planetscale/cli?tab=readme-ov-file#installation) installed on your system to run this command, as the MCP server is shipped as part of your `pscale` client.
+:::
+
+
+<Tabs groupId="interface">
+  <TabItem value="ui" label="goose Desktop" default>
+  <GooseDesktopInstaller
+    extensionId="pscale"
+    extensionName="PlanetScale"
+    description="Invoke the PlanetScale MCP Server to learn more about PlanetScale environments and databases hosted there."
+    command="pscale"
```

Comment:
have you tested this deep link that this generates. I believe the pscale command will fail. 

Sorry to give you the run around, but since this introduces a new command..when someone clicks this deep link, it will fail inside of the goose desktop app. 

See here for allowed commands in the desktop app. I believe you need to add the command to this list . Not sure where else. I believe this list was made for security purposes. https://github.com/block/goose/blob/main/ui/desktop/src/components/settings/extensions/deeplink.ts#L16-L24